assets/javascripts/discourse/mixins/debt-amount-validation.js.es6

```diff
@@ -0,0 +1,26 @@
+import InputValidation from 'discourse/models/input-validation';
+import { default as computed } from 'ember-addons/ember-computed-decorators';
```

Comment:
To import the default you can just do `import computed from 'ember-addons/ember-computed-decorators'` unless there's something specifically different for whatever reason.

Reply:
👍